feature-libs/quote/components/summary/quote-summary.component.spec.ts

```diff
@@ -0,0 +1,136 @@
+import { Component } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { I18nTestingModule } from '@spartacus/core';
+import { Quote, QuoteFacade } from '@spartacus/quote/root';
+import { createEmptyQuote } from '../../core/testing/quote-test-utils';
+import { BehaviorSubject, NEVER, Observable } from 'rxjs';
+import { CommonQuoteTestUtilsService } from '../testing/common-quote-test-utils.service';
+import { QuoteSummaryComponent } from './quote-summary.component';
+
+@Component({
+  selector: 'cx-quote-summary-price',
```

Comment:
```suggestion
  selector: 'cx-quote-summary-prices',
```
Just for consistency